pkg/api/validation/validation.go

```diff
@@ -2024,6 +2024,14 @@ func ValidateHostAliases(hostAliases []api.HostAlias, fldPath *field.Path) field
 	return allErrs
 }
 
+func ValidatePriorityName(priorityName string, fldPath *field.Path) field.ErrorList {
```

Comment:
1. Actually any valid integer is a valid priority, even negative numbers. I am not sure if we can do any particular validation for "Priority".

2. I was not so sure whether we want to allow updating priority or not. I am going to modify ValidatePodUpdate() so that it allows change of priority with the assumption that all updates go through admission checks.